lib/jsdom/living/nodes/HTMLFormElement-impl.js

```diff
@@ -22,6 +25,7 @@ const methods = new Set([
   "dialog"
 ]);
 
+
```

Comment:
I don't think we want the extra line here when it would be inconsistent with the other files.